gremlin-javascript/glv/GraphTraversalSource.template

```diff
@@ -113,6 +113,14 @@ class GraphTraversal extends Traversal {
   constructor(graph, traversalStrategies, bytecode) {
     super(graph, traversalStrategies, bytecode);
   }
+
+  /**
+   * Copy a traversal so as to reset and re-use it.
+   */
+  clone() {
+    return new GraphTraversal(this.graph, this.traversalStrategies, this.getBytecode());
```

Comment:
@spmallette 
This still references the same `graph`, `traversalStrategies`, and `bytecode` of `this`. 
This clones but will affect the cloned and original.
Not sure where to post an issue

Reply:
issues/bugs are tracked in JIRA: https://issues.apache.org/jira/projects/TINKERPOP/